app/controllers/collections_controller.rb

```diff
@@ -46,6 +46,19 @@ def use
     render :json => result, status: :ok
   end
 
+  def collection_relevancy
+    number_of_collections = 20
+
+    data = {
+      query: params[:query],
+      collections: params[:collections][0..number_of_collections - 1],
```

Comment:
Just discussed this with Matthew and Jason. The selected_index still returns, which is good. We are going to add selected_collection to cover cases of a collection selected that is outside the top 20.